libraries/varnish_instance.rb

```diff
@@ -59,7 +60,10 @@ def service_options(resource)
         varnish_args << "-a #{new_resource.listen_address}:#{new_resource.listen_port}"
         varnish_args << "-f #{new_resource.path_to_vcl}"
         varnish_args << "-T #{new_resource.admin_listen_address}:#{new_resource.admin_listen_port}"
-        varnish_args << "-u #{new_resource.user} -g #{new_resource.group}"
+        if new_resource.version == '4.0'
+          # varnish_args << "-u #{new_resource.user} -g #{new_resource.group}"
```

Comment:
This comment should be removed - or better, add a comment explaining why these options are required for Varnish 4.0, but not other versions.